ambient.d.ts

```diff
@@ -246,18 +246,38 @@ declare module '@mojaloop/central-services-shared' {
       TP_CONSENT_PUT_ERROR: string;
     };
   }
+
+  enum EventTypeEnum {
+    AUTHORIZATION = 'AUTHORIZATION',
```

Comment:
why only partial Event.Type wherein source code is the whole list?
In source code lower case notation is used, here uppercase -> it could bring problems in future....
https://github.com/mojaloop/central-services-shared/blob/master/src/enums/events.js#L28